doc/tools/test-tool.md

```diff
@@ -49,15 +49,35 @@ Run dart tests in this package.
 
 Usage: dart_dev test [files or directories...]
 ======== Selecting Tests
--n, --name          A substring of the name of the test to run.
-                    Regular expression syntax is supported.
-                    If passed multiple times, tests must match all substrings.
+-n, --name              A substring of the name of the test to run.
+                        Regular expression syntax is supported.
+                        If passed multiple times, tests must match all substrings.
 
--N, --plain-name    A plain-text substring of the name of the test to run.
-                    If passed multiple times, tests must match all substrings.
+-N, --plain-name        A plain-text substring of the name of the test to run.
+                        If passed multiple times, tests must match all substrings.
 
 ======== Running Tests
--P, --preset        The configuration preset(s) to use.
+-P, --preset            The configuration preset(s) to use.
+    --[no-]release      Build with release mode defaults for builders.
+                        This only applies in projects that run tests with build_runner.
+
+======== Output
+    --reporter          The runner used to print test results.
```

Comment:
Any chance we could include an abbreviation too? 😁

Reply:
> Note: the -r abbreviation was intentionally not added because it conflicts with the abbreviation for the --release flag used when running tests via build_runner. In other words, when running tests via ddev test, the -r abbreviation has no meaning and the full option name (either --release or --reporter) should be used to avoid ambiguity.

Reply:
🤕 I just saw that. 👍

Reply:
I'm open to choosing to support the abbreviation for reporter, but it could lead to confusion if people try to use it to mean `--release`

Reply:
I'm cool with no abbreviation for now.